### ISSUE TITLE
ha snapshot new timeouts and returns code 1

### ISSUE THREAD

**Issue by mathieu-mp** [home-assistant/cli#291]
## Describe the problem

`ha snapshot new`:
- issues a timeout message after 30 seconds 
- then terminates with return code 1 (error)
- at this time, snapshot creation has not ended yet

The issue happened for the first time on today 2.30am CEST. It has never happened before while the scripts runs daily for several months.

## Debug information

1. SSH Addon type

- [X] Community
- [ ] Core
- [ ] Custom

2. Version of the CLI
```
$ ha cli info
update_available: false
version: 2021.04.1
version_latest: 2021.04.1
```

3. Version of Home Assistant Core & the Home Assistant Supervisor

Home Assistant OS: 5.13
Home Assistant Core: 2021.4.6
Home Assistant Supervisor: 2021.04.0

4. Output from executing the command with debug info

`date ; ha snapshot new --no-progress --log-level debug ; echo $?; date`
```txt
Sat May  1 19:28:13 CEST 2021
DEBU[0000] Adding homedir to searchpath                  homedir=/root
INFO[0000] No configfile found
DEBU[0000] Debug flags                                   apiToken=50e46323xxxxxx8b94c076675bb3dc7d193cc2ca6e1c335cba8c1 cfgFile= endpoint=supervisor logLevel=debug noProgress=true rawJSON=false
DEBU[0000] snapshots new                                 args="[]"
DEBU[0000] addons                                        addons="[]"
DEBU[0000] folders                                       folders="[]"
DEBU[0000] [GenerateURI]                                 base=supervisor command=new/full section=snapshots
DEBU[0000] [GenerateURI] Result                          uri="http://supervisor/snapshots/new/full" url="http://supervisor/snapshots/new/full" url(string)="http://supervisor/snapshots/new/full"
Post "http://supervisor/snapshots/new/full": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
1
Sat May  1 19:28:43 CEST 2021
```

## Expected behavior

`ha snapshot new` should:
- terminate after snaphshot is fully created 
- without timeout message
- with return code 0

...to allow correct execution of backup scripts

## Additional context

Anything related to Pull Request #290 ?

**Comment by ikifar2012** [comment]
I am having this issue in my addon as well

**Comment by lukasleitsch** [comment]
I can also confirm this issue. My Backup-Script using a ssh connection and die CLI to create a new snapshot. The script fails due the timeout issue of the CLI command.

Could we provide more information to help debugging this issue?

**Comment by agners** [comment]
This is solved by #296.
